MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -1099,6 +1108,10 @@
 	addMTLVertexFormatCapabilities( macOS_GPUFamily1_v3, UChar4Normalized_BGRA, Vertex );
 }
 #endif
+#if MVK_TVOS
+void MVKPixelFormats::modifyMTLFormatCapabilities(id<MTLDevice> mtlDevice) {
+}
```

Comment:
Not sure what to put in here for pixel and vertex format capabilities..

Reply:
That's also in the document I linked.

Reply:
Have these been validated against the Apple Feature Set docs @cdavis5e identified? 

Probably similar to iOS I should think.

Reply:
I copy/pasted the code from iOS and decremented the family by one. Maybe not the best approach so an audit would be helpful.

Reply:
I audited these against the PDF and fixed everything to match.

Reply:
I am still missing `addMTLVertexFormatCapabilities` calls in this function. I can't figure out where the iOS entries came from? /cc @cdavis5e